tuna/rocmlir/load_job.py

```diff
@@ -69,19 +76,21 @@ def add_jobs(args, dbt):
                    .filter(dbt.session_table.valid == 1,
                            dbt.session_table.id == args.session_id)
     reasons = query.all()
+    if not reasons:
```

Comment:
I could easily mess up the session while debugging, and wanted to catch the broken ones early.